include/mbedtls/mbedtls_config.h

```diff
@@ -3160,6 +3158,9 @@
  * \warning MBEDTLS_SHA512_USE_A64_CRYPTO_IF_PRESENT cannot be defined at the
  * same time as MBEDTLS_SHA512_USE_A64_CRYPTO_ONLY.
  *
+ * \warning This option is experimental. For time being, we can not guarantee
```

Comment:
This (and others) have been in the code for nearly a year, so we shouldn't be suddenly saying they are experimental

Reply:
Will remove it. 

I proposal change it. But it hasn't been approved. I agree do not mark it.

Reply:
It's reasonable to mark somewhere (but not sure here is the right place) that we don't test these on the CI. But "experimental" status is slightly different. We have done a lot of testing on this code, and had confidence in the initial release. The lack of testing on the CI means we have to do a lot of manual testing for any changes - it makes us less confident in modifications to the code from the original

Reply:
> The lack of testing on the CI means we have to do a lot of manual testing for any changes.

I am try to improve that. And I prefer continue this topic in #7004